firmware/projects/Demo/AnalogPWM/main.cc

```diff
@@ -0,0 +1,57 @@
+#include "bindings.h"
+
+using namespace bindings;
+
+/**
+ * @brief Tests the PWMOutput class using various frequencies and duty cycles.
+ */
+void DemoPWM() {
+    pwm.Start();
+    DelayMS(1000);
+
+    pwm.SetFrequency(100);
+    pwm.SetDutyCycle(50.);
+    DelayMS(1000);
+
+    pwm.GetFrequency();
+    pwm.GetDutyCycle();
+    DelayMS(1000);
+
+    pwm.SetFrequency(100);
+    DelayMS(1000);
+
+    pwm.GetFrequency();
+    DelayMS(1000);
+
+    pwm.SetFrequency(-5);
+    pwm.GetFrequency();
+    DelayMS(1000);
+
+    pwm.GetFrequency();
+    DelayMS(1000);
+
+    pwm.Stop();
+}
+
+/**
+ * @brief Tests the AnalogOutput class using various voltages.
+ */
+void DemoAnalog() {
```

Comment:
This method doesn't do much on STM since it cannot output 50V. Can you change the values to be within 0 and 3.3 V, maybe add a few more values